sklearn/utils/tests/test_extmath.py

```diff
@@ -609,7 +609,7 @@ def naive_mean_variance_update(x, last_mean, last_variance,
             _incremental_mean_and_var(A1[i, :].reshape((1, A1.shape[1])),
                                       mean, var, n)
     assert_array_equal(n, A.shape[0])
-    assert_array_almost_equal(A.mean(axis=0), mean)
+    assert_allclose(A.mean(axis=0), mean, rtol=1e-12)
     assert_greater(tol, np.abs(stable_var(A) - var).max())
```

Comment:
this test comes from https://github.com/agramfort/scikit-learn/commit/6a5a2f79603350781ec32b583f7969bda072ddce by @giorgiop